requirements.txt

```diff
@@ -1,8 +1,19 @@
+astroid==2.0.4
+dj-database-url==0.5.0
 Django==2.1
 django-cors-middleware==1.3.1
 django-environ==0.4.5
-psycopg2-binary==2.7.5
 django-extensions==2.1.2
 djangorestframework==3.8.2
+gunicorn==19.9.0
+isort==4.3.4
+lazy-object-proxy==1.3.1
+mccabe==0.6.1
+psycopg2==2.7.5
+psycopg2-binary==2.7.5
 PyJWT==1.4.2
+pylint==2.1.1
```

Comment:
@pngg please remove non-essential requirements eg pylint.

Reply:
@parseendavid we have removed the unnecessary packages